src/super_gradients/recipes/coco2017_yolox.yaml

```diff
@@ -0,0 +1,46 @@
+# YoloX Detection training on CoCo2017 Dataset:
+# YoloX trained in 640x640 mAP@0.5-0.95 (COCO API, confidence 0.01, IoU threshold 0.65, test on 640x640 images)
+# Checkpoints + tensorboards: https://deci-pretrained-models.s3.amazonaws.com/yolox_coco/
+# Recipe runs with batch size = 16 X 8 gpus = 128.
+# Instructions:
+# python -m torch.distributed.launch --nproc_per_node=8 --config-name=coco2017_yolox architecture=yolox_<YOLOX_SIZE> dataset_params.data_dir=<YOUR_COCO_LOCAL_PATH> ckpt_root_dir=<CHEKPOINT_DIRECTORY>
+# Training times and accuracies:
+# yolox_n: 1d 16h 33m 9s on 8 NVIDIA GeForce RTX 3090, mAP: 26.26
```

Comment:
Did you use the same batch size for all models? 
Also, same resolution? Nano and Tiny were originally trained with 416, and when tested in 640 they reach:
- yolox_n mAP: 27.6
- yolox_t mAP: 34.7

Also, are these mAPs with 0.01? If yes then it's really cool, it means that the results are even higher. YOLOX readme reports results with 0.001

Reply:
trained with 640, and yes you: 0.65 and conf 0.01 (with old DetectionMetrics)

Reply:
And yes, same batch size for all.

Reply:
Great, thanks